vim/merlin/autoload/merlin.py

```diff
@@ -88,6 +88,9 @@ def concat_map(f, args):
 
 def current_context():
     filename = vim.eval("expand('%:p')")
+    if platform == "cygwin":
+        cygpath = os.popen('cygpath -m /').read().strip()
+        filename = cygpath + filename
```

Comment:
If you are calling `cygpath` you can pass it the full filename instead of concatenating: `os.popen('cygpath -m ' + filename)` should be enough.